kernel/src/device/fb.rs

```diff
@@ -311,6 +272,7 @@ impl FbHandle {
     }
 
     /// Handles the [`IoctlCmd::GETFSCREENINFO`] ioctl command.
+    /// Takes no input and outputs a [`FbFixScreenInfo`] structure with hardware-specific information.
```

Comment:
Move this to the correct commit.

I think the previous format is okay, perhaps it's actually clearer than the current format:
```rust
    /// Handles the [`IoctlCmd::GETFSCREENINFO`] ioctl command.
    ///
    /// Arguments:
    ///  - Input: [`FbFixScreenInfo`].
    ///  - Output: None.
```